src/util.ts

```diff
@@ -627,3 +627,35 @@ export function checkNotWindows11() {
     );
   }
 }
+
+export const ML_POWERED_JS_QUERIES_PACK_NAME =
+  "codeql/javascript-experimental-atm-queries";
+
+/**
+ * Get information about ML-powered JS queries to populate status reports with.
+ *
+ * This will be:
+ *
+ * - The version string if the analysis will use a specific version of the pack
+ * - "latest" if the analysis will use the latest version of the pack
+ * - "false" if the analysis won't run any ML-powered JS queries
+ * - "multiple" if the analysis will run multiple ML-powered JS query packs (this is an unsupported
+ *   scenario)
+ *
+ * This function lives here rather than in `init-action.ts` so it's easier to test, since tests for
+ * `init-action.ts` would each need to live in their own file. See `analyze-action-env.ts` for an
+ * explanation as to why this is.
+ */
+export function getMlPoweredJsQueriesStatus(config: Config): string {
+  const mlPoweredJsQueryPacks = (config.packs.javascript || []).filter(
+    (pack) => pack.packName === ML_POWERED_JS_QUERIES_PACK_NAME
+  );
+  switch (mlPoweredJsQueryPacks.length) {
+    case 1:
+      return mlPoweredJsQueryPacks[0].version || "latest";
+    case 0:
+      return "false";
+    default:
+      return "multiple";
```

Comment:
I'm pretty sure that the action will happily run each version independently (but I haven't tried this out to be sure). Would it make more sense to do this:

```suggestion
      return mlPoweredJsQueryPacks.map(p => p.version).join(,);
```

Case 1 and the default could be combined.

Reply:
We want to keep the cardinality of this field down.  In practice we can go a bit further than this — it would be sufficient to just capture a fixed set of version strings (i.e. those included in `addBuiltinSuiteQueries` in `config-utils.ts`.  I've pushed a commit that makes this change, and adds some comments explaining (a) that we're keeping the cardinality low and (b) why this is.